lib/JSON/Validator.pm

```diff
@@ -1260,6 +1267,23 @@ block should return C<undef> on success and an error string on error:
 
 See L<JSON::Validator::Formats> for a list of supported formats.
 
+=head2 recursive_data_protection
+
+  my $jv = $jv->recursive_data_protections( $scalar );
+  my $current_state = $jv->recursive_data_protection;
```

Comment:
Use $boolean instead of $scalar/$current_state to indicate that this is a true/false value.

Reply:
Switched to $boolean as suggested